includes/Renderer/Story/Embed.php

```diff
@@ -92,7 +92,11 @@ public function render( array $args = [] ): string {
 		$title  = $this->story->get_title();
 		$poster = ! empty( $this->story->get_poster_portrait() ) ? esc_url_raw( $this->story->get_poster_portrait() ) : '';
```

Comment:
Now we can use `esc_url` instead of `esc_url_raw`.

But I would also suggest to late escape further down before actual output.

```suggestion
		$poster = ! empty( $this->story->get_poster_portrait() ) ? $this->story->get_poster_portrait() : '';
```